app/seaf-cli

```diff
@@ -456,11 +456,12 @@ def seaf_download(args):
     email = tmp['email']
     repo_name = tmp['repo_name']
     version = tmp.get('repo_version', 0)
+    repo_salt = tmp['salt']
 
     more_info = None
     base_url = get_base_url(url)
-    if base_url:
-        more_info = json.dumps({'server_url': base_url})
+    if base_url or repo_salt:
+        more_info = json.dumps({'server_url': base_url, 'repo_salt': repo_salt})
```

Comment:
If repo_salt is None, you should not add it to more_info. The current seaf-daemon code may not be able to handle a repo_salt attribute set to NULL.